Jenkinsfile

```diff
@@ -105,10 +107,16 @@ pipeline {
                     }
                 }
                 stage("Run virtual drive tests") {
+                    environment {
+                        GIT_SSH_COMMAND = 'ssh -i C:/id_rsa -o StrictHostKeyChecking=no'
+                    }
                     steps {
-                        bat """
-                            tox -e py${DEFAULT_TOX_PYTHON_VERSION} -- -m virtual --protocol virtual --junitxml=pytest_reports\\pytest_virtual_report.xml
-                        """
+                        withCredentials([sshUserPrivateKey(credentialsId: 'Bitbucket SSH', keyFileVariable: 'KEY')]) {
+                            bat """
+                                COPY %KEY% C:\\id_rsa
+                                tox -e virtual -- -m virtual --protocol virtual --junitxml=pytest_reports\\pytest_virtual_report.xml
```

Comment:
Since there is a specific tox env for the test with the virtual drive, you could move the `-m virtual --protocol virtual` arguments to the tox file.

Reply:
Done